Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -611,6 +614,8 @@ - (void)configureInstanceIDWithOptions:(FIROptions *)options {
   self.fcmSenderID = GCMSenderID;
   self.firebaseAppID = options.googleAppID;
 
+  [self updateFirebaseInstallationID];
```

Comment:
This method currently triggers [FIRInstanceID didCompleteConfigure] which will get a IID and token in that process. I wonder if this branch is still the same?
In that case this method might not be needed we can just set the installationID from didCompleteConfigure method.

Reply:
As far as I can see `[self didCompleteConfigure];` isn't called for extensions. I'm no sure if it's an issue though. I would prefer to keep it independent, because only the first FID request is relatively expensive and the consequent ones are chip because of the caching.

Reply:
it is not called for extensions, but that's just to keep app extension less expensive so we don't have to worry about it for now.

Let's keep it independent then as that's an old function and is as always overly complicated.